wannierberri/__parallel.py

```diff
@@ -48,6 +47,18 @@ def __init__(self,
             self.pool_K,self.npar_K=pool(0)
         elif self.method == "ray" : 
             ray_init_loc={}
+            if cluster:
+
+                def set_opt(opt,def_val):
+                    if opt not in ray_init:
+                        ray_init_loc[opt] = def_val()
+                    else:
+                        print (f"WARNING: the ray cluster will use '{ray_init[opt]}' provided in ray_init")
+
+                set_opt('address'          , lambda : 'auto')
+                set_opt('_node_ip_address' , lambda : os.environ["ip_head"].split(":")[0])
+                set_opt('_redis_password'  , lambda : os.environ["redis_password"])
```

Comment:
Why did you use lambda function instead of just passing a variable?

Reply:
added a comment in the code:

```
                # The follwoing is done for testing, when __init__ is called with `cluster = True`,
                # but no actual ray cluster was initialized (and hence the needed environmental variables are not set
```